src/views/apply/dashboard.html

```diff
@@ -0,0 +1,104 @@
+{% extends 'default.html' %}
+{% set title = 'Apply for Hack Cambridge' %}
+{% set is_dashboard = true %}
+{% block pageHeading %}Dashboard{% endblock %}
+{% block pageContent %}
+<header class="content-wrapper-center">
+  <h3>Hello, {{ user.first_name }}!</h3>
+</header>
+
+  {# The main status banner #}
+  <div class="content-wrapper-center">
+    {% if applicationStatus == 'incomplete' %}
+      <div class="status-message warning">
+        <h2>Your application is incomplete &nbsp; &#128533;</h2>
+        <span>Finish it off by fixing whatever's incomplete below</span>
+      </div>
+    {% elseif applicationStatus == 'in-review' %}
+      <div class="status-message success">
+        <h2>Your application is being reviewed &nbsp; &#9989;</h2>
```

Comment:
http://caniemoji.com/ seems to indicate that we'll be mostly fine. But we should probably get a feel for what other OS' emoji look like as well